build/management/commands/build_g_protein_structures.py

```diff
@@ -262,6 +281,37 @@ def handle(self, *args, **options):
                         except Exception as msg:
                             self.logger.error("SignprotStructure of {} {} failed\n{}: {}".format(a.entry_name, pdb, type(msg), msg))
 
+        if options["debug"]:
+            print(datetime.datetime.now() - startTime)
+
+    @classmethod
```

Comment:
For here, but also several other spots:
The class is not used in the function, so shouldn't this be a staticmethod instead of a classmethod?